Spec/RealtimeClientConnection.swift

```diff
@@ -3028,26 +2989,165 @@ class RealtimeClientConnection: QuickSpec {
                     default:
                         fail("unhandled: \(fixtureMessage["expectedType"].string!)")
                     }
+                }
 
-                    waitUntil(timeout: testTimeout) { done in
-                        channel.publish([message]) { err in
-                            if let err = err {
-                                fail("\(err)")
+                // https://github.com/ably/wiki/issues/22
+                it("should encode and decode fixture messages as expected") {
+                    let options = AblyTests.commonAppSetup()
```

Comment:
Perhaps I am misunderstanding still, but in these tests, at what point do we publish with a raw JSON message (i.e. getting the fixture into Ably in a guaranteed portable way) and get that message over a JSON transport and confirm the `message.data` is as expected then?